src/SkynetServer/Network/Packets/P27ProfileImage.cs

```diff
@@ -1,30 +1,26 @@
 ﻿using SkynetServer.Model;
 using SkynetServer.Network.Attributes;
+using SkynetServer.Sockets;
 using System;
 using System.Collections.Generic;
 using System.Text;
-using System.Threading.Tasks;
-using VSL;
 
 namespace SkynetServer.Network.Packets
 {
-    [Message(0x27, PacketPolicy.Duplex)]
-    [RequiredFlags(MessageFlags.FileAttached)]
-    [AllowedFlags(MessageFlags.Unencrypted | MessageFlags.FileAttached)]
-    internal sealed class P27ProfileImage : P0BChannelMessage
+    [Packet(0x27, PacketPolicies.Duplex)]
```

Comment:
[OT] I am so far down the review page that syntax highlighting stopped working [/OT]